CONTRIBUTING.md

```diff
@@ -84,7 +84,7 @@ Before submitting an enhancement suggestion, check if there is already a filed o
 First of all, you can validate it with the community, our official channels are:
 
 - [Discord](https://discord.gg/hemixyz)
-- [Twitter](https://twitter.com/hemi_xyz)
+- [Twitter](https://x.com/hemi_xyz)
```

Comment:
The name should be changed too.

```suggestion
- [X](https://x.com/hemi_xyz)
```

Reply:
Changes applied as per your suggestion. Please review.

Reply:
@gabmontes Hi! Just wanted to kindly follow up on this PR. Please let me know if there's anything else needed from my side to proceed with the merge. 😊